### ISSUE TITLE
I am using vi mode and the green I and red N show up above so I have a three line prompt

### ISSUE THREAD

**Issue by matthiasdebernardini** [IlanCosman/tide#26]
Fish version 3.1.2 on Ubuntu 18

I switched to tide because pure was not handling tmux correctly (the prompt kept printing weird characters). 

However below you can see it causes a three line prompt - how can I make that go away but keep vi mode?

Pure handles vi mode by switching reversing the direction of the arrow depending on if youre in insert mode or normal mode - its really minimal and I like it. 

Is it possible to add something like this?

<img width="61" alt="image" src="https://user-images.githubusercontent.com/32387851/93201871-17638b00-f752-11ea-93da-3f9d92ab7f72.png">

**Comment by IlanCosman** [comment]
For sure; I'm working on it now. One issue to consider is that Tide offers multiple styles. I'll try to find a way to make this work for both the minimal and powerline styles.

**Comment by IlanCosman** [comment]
Hi @matthiasdebernardini, can you try installing Tide from the `vi_mode` branch and playing around with it?

```console
curl -sL git.io/tide | source && tide_install vi_mode
```

**Comment by matthiasdebernardini** [comment]
sure, but do I have to uninstall it first?

**Comment by IlanCosman** [comment]
No,

**Comment by matthiasdebernardini** [comment]
TLDR: works well so far

This is all with tmux btw

I am trying to add as much info that you might find interesting.
<img width="1790" alt="Screenshot 2020-09-21 at 13 30 42" src="https://user-images.githubusercontent.com/32387851/93761768-a66b1a00-fc0e-11ea-9efb-dff801b1ea70.png">

`❯ curl -sL git.io/tide | source && tide_install vi_mode
Installing tide theme...
set: Erase needs a variable name

~/.config/fish/tide_theme/configure/choices/all/finish.fish (line 24):
        set -e fake_tide_right_prompt_items[(contains --index 'vi_mode' $fake_tide_right_prompt_items)] # No need for vi_mode if prompt_char is activated
        ^
in function '_tide_finish'
        called on line 39 of file -
in function '_tide_actual_install' with arguments 'vi_mode'
        called on line 23 of file -
in function 'tide_install' with arguments 'vi_mode'

(Type 'help set' for related documentation)
Tide theme installed!
`

`Choice [y/n] y
set: Erase needs a variable name

~/.config/fish/tide_theme/configure/choices/all/finish.fish (line 24):
        set -e fake_tide_right_prompt_items[(contains --index 'vi_mode' $fake_tide_right_prompt_items)] # No need for vi_mode if prompt_char is activated
        ^
in function '_tide_finish'
        called on line 14 of file ~/.config/fish/tide_theme/configure/choices/all/finish.fish
in function 'finish'
        called on line 33 of file ~/.config/fish/functions/_tide_sub_configure.fish
in function '_next_choice' with arguments 'all/finish'
        called on line 21 of file ~/.config/fish/tide_theme/configure/choices/all/prompt_spacing.fish
in function 'prompt_spacing'
        called on line 33 of file ~/.config/fish/functions/_tide_sub_configure.fish
in function '_next_choice' with arguments 'all/prompt_spacing'
        called on line 16 of file ~/.config/fish/tide_theme/configure/choices/all/prompt_connection_andor_frame_color.fish
in function 'prompt_connection_andor_frame_color'
        called on line 33 of file ~/.config/fish/functions/_tide_sub_configure.fish
in function '_next_choice' with arguments 'all/prompt_connection_andor_frame_color'
        called on line 21 of file ~/.config/fish/tide_theme/configure/choices/pure/pure_prompt_height.fish
in function 'pure_prompt_height'
        called on line 33 of file ~/.config/fish/functions/_tide_sub_configure.fish
in function '_next_choice' with arguments 'pure/pure_prompt_height'
        called on line 21 of file ~/.config/fish/tide_theme/configure/choices/pure/pure_show_time.fish
in function 'pure_show_time'
        called on line 33 of file ~/.config/fish/functions/_tide_sub_configure.fish
in function '_next_choice' with arguments 'pure/pure_show_time'
        called on line 19 of file ~/.config/fish/tide_theme/configure/choices/pure/pure_nonperm_content_location.fish
in function 'pure_nonperm_content_location'
        called on line 33 of file ~/.config/fish/functions/_tide_sub_configure.fish
in function '_next_choice' with arguments 'pure/pure_nonperm_content_location'
        called on line 34 of file ~/.config/fish/tide_theme/configure/choices/all/style.fish
in function 'style'
        called on line 33 of file ~/.config/fish/functions/_tide_sub_configure.fish
in function '_next_choice' with arguments 'all/style'
        called on line 27 of file ~/.config/fish/functions/_tide_sub_configure.fish
in function '_tide_begin'
        called on line 23 of file ~/.config/fish/functions/_tide_sub_configure.fish
in function '_tide_sub_configure'
        called on line 16 of file ~/.config/fish/functions/tide.fish
in function 'tide' with arguments 'configure'
        called on line 54 of file -
in function '_tide_actual_install' with arguments 'vi_mode'
        called on line 23 of file -
in function 'tide_install' with arguments 'vi_mode'

(Type 'help set' for related documentation)
`
<img width="102" alt="Screenshot 2020-09-21 at 13 34 56" src="https://user-images.githubusercontent.com/32387851/93762113-3f019a00-fc0f-11ea-8910-b9d08e1e4a26.png">
<img width="102" alt="Screenshot 2020-09-21 at 13 35 07" src="https://user-images.githubusercontent.com/32387851/93762129-4628a800-fc0f-11ea-992b-03fc5fedc16a.png">

love it!


Thanks so much for quickly patching this - this is a great prompt.

**Comment by IlanCosman** [comment]
Oh, yeah, all of those errors are from using the install script while Tide is already installed. Thanks a ton for opening this issue and helping me test it 😄
